tests/format/js/arrows/with-inner-spacing/__snapshots__/jsfmt.spec.js.snap

```diff
@@ -957,23 +1101,24 @@ export const bem = block =>
  * @param {String} block - the BEM Block you'd like to select.
  * @returns {Function}
  */
-export const bem = block =>
+export const bem =
+  block =>
   /**
    * @param {String} [element] - the BEM Element within that block; if undefined, selects the block itself.
    * @returns {Function}
    */
   element =>
```

Comment:
The indentation in this arrow function block does not look right to me.

Reply:
From a cursory glance, it looks like this indentation is consistent with Prettier 2.3.1's behaviour: https://github.com/prettier/prettier/blob/2.3.1/tests/format/js/arrows/__snapshots__/jsfmt.spec.js.snap#L2110

Reply:
You are right, aargh!

I think we should raise an issue on Prettier, if it was not reported before, and see if we can contribute a bug fix. I would be happy to do this if you like.

Reply:
Looks like the change in behaviour came in prettier/prettier#9992: https://github.com/prettier/prettier/commit/046ffb11f980269937b2e58a30567efdcdbf230c#diff-9fc5f6234a69ade9849cab79fc6c92588186225a105330442778daf68403315bL1836-R2051

There's a related issue, prettier/prettier#11001, regarding print-width, but I don't see much discussion regarding indentation.